charts/zookeeper/templates/zookeeper.yaml

```diff
@@ -105,9 +105,11 @@ spec:
     terminationGracePeriodSeconds: {{ .Values.pod.terminationGracePeriodSeconds }}
     {{- end }}
     serviceAccountName: {{ default "zookeeper" .Values.pod.serviceAccountName }}
-    {{- if .Values.pod.imagePullSecrets }}
+    {{- if or .Values.global.imagePullSecrets .Values.pod.imagePullSecrets }}
     imagePullSecrets:
-{{ toYaml .Values.pod.imagePullSecrets | indent 6 }}
+    {{- range (default .Values.global.imagePullSecrets .Values.pod.imagePullSecrets) }}
+      - name: {{ . }}
+    {{- end }}
```

Comment:
This would constitute a breaking change. Do we have grounds for it?
The schema for `.Values.pod.imagePullSecrets` seems to require an array of key-value pairs, while proposed schema for `.Values.global.imagePullSecrets` seems to an array of strings.